src/bin/cargo-miri.rs

```diff
@@ -256,7 +258,7 @@ path = "lib.rs"
         show_error(format!("Failed to run xargo"));
     }
 
-    // That should be it!  But we need to figure out where xargo built stuff.
+    // That should be it! But we need to figure out where xargo built stuff.
```

Comment:
I have no opinion on those, but @RalfJung added them since their editor does this automatically. If in doubt, I'd opt for not changing anything

Reply:
On the original discussion on the rustc thread, I think we decided these were archaic (a remnent of typewriter typesetting mainly), but if you want to adopt a different convention here...

Reply:
"archaic", that's funny. ;)  emacs inserts these, but I don't actually use emacs for Rust coding. My muscle memory adds them.  Also, if you read properly typeset text today, there's more space after a full stop than between two words -- this is just emulating that. Jut sure how that's "archaic", other than "MS Word doesn't do it" (but that says more about MS Word than anything else...)

I don't care if you remove them or keep them. I'll probably add them back even if I try not to.

Reply:
@RalfJung Yeah, I didn't express myself right, above. I was just taught it was old-fashioned, but the truth is more nuanced. In another comment, I clarified that modern typesetting does indeed (usually) add more space than a word-separating space after a full stop, but doesn't require you to *type* two spaces (it's also not as much as too spaces). So when you have a monospaced font without proper typesetting... I guess it's an open question!

Thanks for not minding. I guess if you keep using them thanks to muscle memory, it's no big deal. :-)